views/layouts/main.hbs

```diff
@@ -67,6 +67,11 @@
 						Events
 					</a>
 				</li>
+				<li>
```

Comment:
This will possibly also need adding in the non-admin nav menu

Reply:
Yes, it will once I've worked out the above